aDzbanuszek/people/Teacher.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+import os
+from aDzbanuszek.people.Person import Person
+
+
+class Teacher(Person):
+
+    def __init__(self, name, second_name, born_data, vacationDay, engagement):
+        super(Teacher, self).__init__(name, second_name, born_data)
+        self.vacationDay = int(vacationDay)
```

Comment:
w tym wypadku nie ma potrzeby rzutowania - jesli ktos poda stringa to wywali sie funkcja takeVacation. Nie jest to niepoprawne, ale mysle ze zbedne